doc/getting_started/index.rst

```diff
@@ -471,71 +471,142 @@ Install a Toolchain
 A toolchain provides a compiler, assembler, linker, and other programs required
 to build Zephyr applications.
 
+The Zephyr Software Development Kit (SDK) contains toolchains for each of
```

Comment:
I thought we were going to stop calling this the SDK? Did I miss a memo?

Reply:
As per the discussion in the TSC, we will stick with the `Zephyr SDK` nomenclature for now.